config/webpack.config.main.prod.ts

```diff
@@ -0,0 +1,17 @@
+import TerserPlugin from 'terser-webpack-plugin';
+import type webpack from 'webpack';
+import { merge } from 'webpack-merge';
+import baseConfig from './webpack.config.main.base';
+
+const configuration: webpack.Configuration = {
+  devtool: 'source-map',
```

Comment:
Do we need this in the prod one?

Reply:
🤷‍♂️

Reply:
IMO source-maps make sense if we have a centralized error reporting. That's probably why the template has it. Since we don't have exception tracking, there's nowhere to send the source maps, so I don't think it's necessary.